src/python/bcc/table.py

```diff
@@ -397,6 +397,32 @@ def clear(self):
         for k in self.keys():
             self.__delitem__(k)
 
+
+    def items_lookup_and_delete_batch(self):
+        # batch size is set to the maximum
+        batch_size = self.max_entries
+        in_batch = ct.c_void_p(0)
+        out_batch = ct.c_void_p(0)
+        keys = (type(self.Key()) * batch_size)()
+        values = (type(self.Leaf()) * batch_size)()
+        count = ct.c_uint32(batch_size)
+        res = lib.bpf_lookup_and_delete_batch(self.map_fd,
+                                              ct.byref(in_batch),
```

Comment:
We should pass a NULL pointer for in_batch. I think `ct.byref(in_batch)` here is a pointer to value 0? I see some document saying `None` can be represented as NULL pointer. But I did not do experiments, so not sure.